### DISCUSSION TITLE
print menu

### DISCUSSION THREAD

**Discussion by 3dfixer** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1508] [Q&A] [answered]
hi all, can anyone tell me what the brief button on the print menu is for?

**Comment by kisslorand** [comment, marked as answer]
That button brings up a popup with the statistics of the last print. If statistics info were embedded into the gCode than you'll have the file name, print time, filament length used, filament weight used and the cost of the filament that was used.
If the filament data was not embedded into the gCode file than you will have the filename, print time and the filament length used.
You will find more detailed info at the bottom of the page: https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware

**Reply by 3dfixer** [reply]
is this a marlin, tft or slicer thing?

**Reply by kisslorand** [reply]
All three. The slicer includes the statistics data into the gCode, Marlin sends it to the TFT, TFT stores and displays it.
